step.yml

```diff
@@ -6,124 +6,125 @@ website: https://github.com/bitrise-steplib/steps-export-xcarchive
+    is_required: true
 
-        - `1MZX23ABCD4`
-  - product: "app"
-    opts:
-      title: "Select a product to distribute"
-      summary: ""
-      description: |-
-        Describes which product to export.   
-        Possible options are App or App Clip.
-      value_options:
-      - "app"
-      - "app-clip"
-      is_required: true
-  - custom_export_options_plist_content:
-    opts:
-      title: "Custom export options plist content"
-      description: |-
-        Specifies a custom export options plist content that configures archive exporting.
-        If empty, step generates these options based on the embedded provisioning profile,
-        with default values.
+- distribution_method:
+  opts:
+    title: "Distribution method"
+    summary: "Describes how Xcode should export the archive."
+    description: |-
+      Describes how Xcode should export the archive.
+    value_options:
+    - "development"
+    - "app-store"
+    - "ad-hoc"
+    - "enterprise"
+    is_required: true
 
-        Auto generated export options available for export methods:
+# IPA export configuration
 
-        - app-store
-        - ad-hoc
-        - enterprise
-        - development
+- export_development_team:
+  opts:
+    category: "IPA export configuration"
+    title: "Developer Portal team"
+    summary: "The Developer Portal team to use for this export"
+    description: |-
+      The Developer Portal team to use for this export.
 
-        If step doesn't find export method based on provisioning profile, development will be use.
+      Format example: `1MZX23ABCD4`
+
+- compile_bitcode: "yes"
+  opts:
+    category: "IPA export configuration"
+    title: "Rebuild from bitcode"
+    summary: "For __non-App Store__ exports, should Xcode re-compile the app from bitcode?"
+    description: |-
+      For __non-App Store__ exports, should Xcode re-compile the app from bitcode?
+    value_options:
+    - "yes"
+    - "no"
+    is_required: true
+
+- upload_bitcode: "yes"
+  opts:
+    category: "IPA export configuration"
+    title: "Include bitcode"
+    summary: "For __App Store__ exports, should the package include bitcode?"
+    description: |-
+      For __App Store__ exports, should the package include bitcode?
+    value_options:
+    - "yes"
+    - "no"
+    is_required: true
+
+- custom_export_options_plist_content:
+  opts:
+    category: "IPA export configuration"
+    title: "Export options plist content"
+    summary: "Specifies a plist file content that configures archive exporting."
+    description: |-
+      Specifies a custom export options plist content that configures archive exporting.
+      If empty, step generates these options based on the embedded provisioning profile,
+      with default values.
+
+      Call `xcodebuild -help` for available export options.
+
+# Debugging
+
+- verbose_log: "yes"
+  opts:
+    category: "Debugging"
+    title: "Enable verbose logging?"
+    description: Enable verbose logging?
```

Comment:
summary: If this input is set, the Step will print additional logs for debugging.